docs/reference/changelog.md

```diff
@@ -24,6 +24,9 @@
     * 'waterManagement'
   * [#493](https://github.com/open-contracting/infrastructure/pull/493) - projectType:
     * 'maintenance'
+  * [#494](https://github.com/open-contracting/infrastructure/pull/494) - policyAlignment:
```

Comment:
Let's sort in alphabetical order for easier reference, so policyAlignment before projectType.